fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraVersioning.java

```diff
@@ -191,4 +195,14 @@ protected FedoraResource resource() {
     protected Session session() {
         return session;
     }
+
+    /**
+     * Determines if a label contains valid characters
+     * @param label label
+     * @return whether label contains valid characters
+     */
+    public boolean invalidLabel(final String label) {
```

Comment:
This method should be "private".
Also, I agree with the javadocs on this method, that it is more clear if the method returns whether the arg label is valid. Additionally, it would make the logic above easier to read, i.e. no double-negative. Instead it would read:
`if (!isBlank(slug) && validLabel(slug)) {`

I suggest renaming the method and returning `!matcher.find()`